app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadFragment.java

```diff
@@ -280,13 +280,14 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 zimManageActivity.mSectionsPagerAdapter.libraryFragment.libraryAdapter.getFilter().filter(((ZimManageActivity) getActivity()).searchView.getQuery());
               }
             })
-            .setNegativeButton(R.string.no, (dialog, i) -> {
-              if (hasArtificiallyPaused) {
+                         .setNegativeButton(R.string.no, (dialogInterface, i) -> {
```

Comment:
wrong indentation.

Reply:
.setNegativeButton(.................)  is exactly under the 
.setPositiveButton(..................)